core/src/components/select/select.ionic.outline.scss

```diff
@@ -50,3 +50,11 @@
 
   box-sizing: border-box;
 }
+
+// Focus
```

Comment:
I also noticed that the invalid example on the highlight test page wasn't rendering correctly. It should have a red border. This can be seen when comparing it to md.

In order to fix that, the following styles need to be added:
```scss
/**
 * If the select has a validity state, the
 * border should reflect that as a color.
 */
:host(.has-focus.select-fill-outline.ion-valid),
:host(.select-fill-outline.ion-touched.ion-invalid) {
  --border-color: var(--highlight-color);
}
```

Reply:
Hum, didn't notice as it seems to work well when receiving the actual focus

Reply:
Fixed!